projects/vs/psa-boot-test/psa-boot-test.cpp

```diff
@@ -9,8 +9,8 @@ namespace psaboottest
 {
 	TEST_CLASS(psaboottest)
 	{
-		#define GET_TEST_PACKAGE(_pack)				(boot_upgrade_package_t*)copy_package(_pack, sizeof(_pack))
-		#define RELEASE_TEST_PACKAGE(_pack)			free(_pack)
+			#define GET_TEST_PACKAGE(_pack)				(boot_upgrade_package_t*)copy_package(_pack, sizeof(_pack))
```

Comment:
Please do not change unrelated lines, so revert these lines

Reply:
done